api/config/cdr_versions_prod.json

```diff
@@ -22,6 +22,6 @@
     "releaseNumber": 1,
     "numParticipants": 116157,
     "cdrDbName": "cdr20181120",
-    "publicDbName": "public20181120"
+    "publicDbName": "public20181129"
```

Comment:
Did you just generate this? Maybe we can roll this change into my other CDR version update so it can be applied in one go: https://github.com/all-of-us/workbench/pull/1557

Reply:
Yeah i generated just the public data.

Reply:
OK - can I pull this into that PR instead? Unless this change is needed along with this PR?

Reply:
No that's fine. You can pull this out and i am going to revert this change in this pr.

Reply:
SOunds good - added to https://github.com/all-of-us/workbench/pull/1557